lib/features/children/overview/pages/family_overview_page.dart

```diff
@@ -90,14 +95,19 @@ class FamilyOverviewPage extends StatelessWidget {
     return const SizedBox();
   }
 
-  void _addNewChild(BuildContext context, FamilyOverviewUpdatedState state) {
-    AnalyticsHelper.logEvent(
+  void _addNewChild(
+      BuildContext context, FamilyOverviewUpdatedState state) async {
+    unawaited(AnalyticsHelper.logEvent(
       eventName: AmplitudeEvents.addMemerClicked,
+    ));
+    final dynamic result = await Navigator.push(
```

Comment:
I'm assuming this is test code that you don't want to merge yet?

Reply:
indeed that was to see it